packages/compass-aggregations/src/stores/store.js

```diff
@@ -185,12 +44,57 @@ export const setSourceNames = (store, sourceName) => {
  * @returns {Store} The store.
  */
 const configureStore = (options = {}) => {
-  const store = createStore(reducer, applyMiddleware(thunk));
+  const { collection } = toNS(options?.namespace ?? '');
+
+  const store = createStore(
+    reducer,
+    {
+      appRegistry: {
+        localAppRegistry: options.localAppRegistry ?? null,
+        globalAppRegistry: options.globalAppRegistry ?? null
+      },
+      dataService: {
+        error: options.dataProvider?.error ?? null,
+        dataService: options.dataProvider?.dataProvider ?? null
+      },
```

Comment:
Totally not possible to end up here without dataService, but this is what tests are expecting, so I decided to preserve the current behavior